python/foglamp/tasks/north/sending_process.py

```diff
@@ -198,7 +198,7 @@ class SendingProcess(FoglampProcess):
     """ The amount of time the fetch operation will sleep if there are no more data to load or in case of an error """
     TASK_SEND_SLEEP = 0.5
     """ The amount of time the sending operation will sleep in case of an error """
-    TASK_SLEEP_MAX_INCREMENTS = 4
+    TASK_SLEEP_MAX_INCREMENTS = 7
```

Comment:
why its made  4 to 7 ?

Reply:
it seemed better to extend the max increment to a larger value as it starts from a limited amount of sleep time = 0.5 s

Reply:
Okay